### ISSUE TITLE
Forty-four more concept videos, and the run that proves #400 and #401 both landed

### ISSUE THREAD

**Issue by opendroid** [opendroid/the-infinity#413]
The second `--concepts` pass, and the first one where both fixes could be measured against real YouTube.

## #400 is exact, not approximately better

```
predicted   98 targets × 101 units + 10 channel lookups = 9,908
spent                                                   = 9,908
```

Not one `rateLimitExceeded` line in the run, and it ended on `Budget reached at 9908 units` rather than being refused into silence.

| | targets searched | units spent |
|---|---|---|
| first run (before #400) | 41 | 9,951 |
| this run | **98** | 9,908 |

The 58% that was going to requests YouTube had already refused is gone, and the once-per-operation accounting now predicts the spend to the unit.

## #401 is four for four

The `--redo` names from #403, re-queried with the field disambiguator:

| concept | before | after |
|---|---|---|
| `ablation` | AFib cardiac ablation, 0.55 | *Training and Validation Methodology — **Ablation Studies*** |
| `alibi` | a legal-English dictionary entry, 0.54 | Aleksa Gordić — ***ALiBi**: Train Short, Test Long* |
| `adam` | *Biblically Accurate Adam and Eve*, 0.56 | the optimizer, now **rank 1** |
| `active-learning` | classroom pedagogy, 0.56 | Computerphile, now **rank 1** |

Two rescued from nothing, two confirming the hand picks from #411 and promoting them to first. The Genesis video and the cardiac guides are gone from the results entirely.

**The fix is not total, and that is worth recording.** `cold-start` still returns *"⚠️ Cold Start Dangers EXPLAINED"* from an auto-repair channel at 0.56, and `custom-kernel` still returns *The Kernel Trick*. Naming the field narrows the query; it does not disambiguate a term that is a genuine homonym **inside** the field, or one whose ML sense is rarer on YouTube than its everyday sense.

## The picks

44 verified through oEmbed, 0 rejected, out of 85 targets that returned candidates.

The strongest are unambiguous: StatQuest's *Cross Validation* (0.96, 1.5M views), *seq2seq Encoder-Decoder* (0.96), *Decision Trees* (0.81, 1.3M); Yannic Kilcher on CLIP (0.95); Stanford Online on double descent (0.94); Andrew Ng on dropout (0.95); Angelopoulos' own conformal-prediction tutorial.

**Two more picks argue with the score**, the same pattern #411 recorded:

- `deceptive-alignment` → **Robert Miles at 0.40**, 92K views. He is the AI-safety explainer channel and the video is exactly the concept; the scorer only saw low title overlap.
- `convolutional-network` → **deeplizard at 0.41**, 1.5M views.

That is now four times across two batches that the top-scored candidate was not the right one. The score sorts a list for a person to read; it does not decide, and the `--min-score` threshold is a floor on what is worth *reading*, not a verdict.

## What was skipped

41 of 85 targets got nothing:

- **Homonyms the field fix did not reach** — `cold-start` (car engines), `custom-kernel` (the kernel trick), `causal-masking` (causal inference), `code-search`, `compilation-cache`, `data-mixing` (Gaussian mixtures), `dense-retrieval` (RAG), `early-exit` (3B1B's *what is a neural network*, 0.67 and entirely the wrong concept), `deduplication` (storage dedup).
- **Already served by a video about the same thing** — `continual-learning`, `canary-rollout`, `decoder-only-transformer` (the top candidate is the exact video already attached), `conditional-random-field`'s top hit likewise.
- **Essentially unwatched or research talks rather than teaching** — `data-parallelism` (720, 547 views), `counterfactual`, `dp-sgd`, `cross-lingual-transfer`, `elastic-weight-consolidation`, `autotuning`, `batch-inference`.

## Acceptance criteria

- [ ] 44 concept-scoped `kind: video` explainers added, titles and authors exactly as oEmbed reported
- [ ] Concept-scoped entries still render before domain-scoped ones, checked corpus-wide
- [ ] No node gains a second video about the same thing it already had
- [ ] `check:explainers` passes in CI, where `youtube.com` is reachable — that is what confirms the recorded attribution

Refs #61, #384, #386, #400, #401, #403, #411, ADR-0017, ADR-0018.
